dotfiles/custom_env_vars

```diff
@@ -0,0 +1,5 @@
+# This is a 'GramDemolition' File
```

Comment:
save existing `.custom_env_vars` if it exists.
maybe even copy contents to new one

Reply:
Created issue: https://github.com/taksjosephrefugio/GramDemolition/issues/13